vmdb/app/helpers/application_helper.rb

```diff
@@ -2706,6 +2708,7 @@ def render_listnav_filename
     elsif %w(offline retired templates vm vm_cloud vm_or_template).include?(@layout)
       "vm"
     elsif %w(action availability_zone cim_base_storage_extent cloud_tenant condition container_group
+             container_route container_project
```

Comment:
@h-kataria , should the container_route and container_project be also added in line 2702?
I see some entities appear in both the "if" and "elsif" parts..